api/nakadi-event-bus-api.yaml

```diff
@@ -479,7 +475,40 @@ paths:
                 Number of seconds the client ought to wait before making a follow-up request.
               type: number
 
-  /event-types/{name}/partitions:
+  '/event-types/{name}/schemas':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        List of schemas ordered from most recent to oldest.
+    responses:
+        '200':
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/EventTypeSchema'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/event-types/{name}/schemas/{version}':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        Retrieves a given schema version. A special `{version}` key named 'latest' is provided for
+        convenience.
+    responses:
+        '200':
+          description: |
+            Schema as JSON.
+        '401':
```

Comment:
API guidelines now allow to skip the common failure codes (like 401) in swagger file
See here http://zalando.github.io/restful-api-guidelines/http/Http.html#must-provide-error-documentation